docs/metrics.md

```diff
@@ -24,6 +24,48 @@ of the last second of activity, whereas the 1 minute intervals are used for
 reporting values to an external database in which case the values reported are
 all reflective of the last minute of activity.
 
+### Default Metrics
+
+Colossus makes several metrics available in a default service, they are:
+
+* *service name*/system/fd_count
```

Comment:
Also these are only the metrics for a server, clients have their own set of metrics not included here.

Also, this is a nitpick, but remember that `colossus-metrics`is a separate library from colossus itself, so this section doesn't apply for people using colossus-metrics but not colossus.  So maybe change the title to "Default Metrics in Colossus"